--- poedit-1.4.4/install/poedit.desktop.org	2010-01-29 22:46:46.000000000 +0100
+++ poedit-1.4.4/install/poedit.desktop	2010-01-29 22:47:41.000000000 +0100
@@ -2,6 +2,7 @@
 Name=Poedit
 Comment=Gettext catalogs editor
 Comment[fr]=Outil de traduction
+Comment[pl]=Edytor katalogów Gettexta
 MimeType=application/x-po;application/x-gettext;text/x-gettext-translation;text/x-po;
 Exec=poedit %F
 Icon=poedit
