--- poedit-1.3.6/locales/Makefile.am.orig	2007-03-16 09:06:53.000000000 +0000
+++ poedit-1.3.6/locales/Makefile.am	2007-03-16 09:07:33.000000000 +0000
@@ -1,5 +1,5 @@
 
-POEDIT_LINGUAS = af_ZA am ar ast be bg bn br ca cs da de el eo es es_PR et eu fa_IR fi fr fur gl he hi hr hu id is it ja ka ko ku ky lt lv mk mn ms nb nl nn pa pl pt_BR pt_PT ro ru sk sl sq_AL sr sv ta th tr tt uk wa zh_CN zh_TW
+POEDIT_LINGUAS = af am ar ast be bg bn br ca cs da de el eo es es_PR et eu fa fi fr fur gl he hi hr hu id is it ja ka ko ku ky lt lv mk mn ms nb nl nn pa pl pt_BR pt ro ru sk sl sq sr sv ta th tr tt uk wa zh_CN zh_TW
 WXWIN_LINGUAS  = af ca cs da de el es eu fi fr hi hu id it ja nb_NO nl pl pt_BR ru sl sv tr uk zh_CN zh_TW
 
 localedir = $(datadir)/locale
