--- poedit-1.3.9/locales/Makefile.am~	2007-12-12 11:15:14.616977859 +0100
+++ poedit-1.3.9/locales/Makefile.am	2007-12-12 11:16:16.720195702 +0100
@@ -1,5 +1,5 @@
 
-POEDIT_LINGUAS = af_ZA am ar ast be@latin be bg bn br ca cs da de el eo es es_PR et eu fa_IR fi fr fur ga gl he hi hr hu id is it ja ka ko ku ky lt lv mk mn ms nb nl nn pa pl pt_BR pt_PT ro ru sk sl sq_AL sr sv ta th tr tt uk ur wa zh_CN zh_TW
+POEDIT_LINGUAS = af am ar ast be bg bn br ca cs da de el eo es es_PR et eu fa fi fr fur ga gl he hi hr hu id is it ja ka ko ku ky lt lv mk mn ms nb nl nn pa pl pt_BR pt ro ru sk sl sq sr sv ta th tr tt uk ur wa zh_CN zh_TW
 
 localedir = $(datadir)/locale
 
