diff -ru poedit.orig/src/catalog.cpp poedit/src/catalog.cpp
--- poedit.orig/src/catalog.cpp	Tue Aug 21 12:52:16 2001
+++ poedit/src/catalog.cpp	Sat Nov 10 21:17:51 2001
@@ -230,7 +230,11 @@
     wxCSConv conv(!!charset ? charset : wxString(_T("utf-8")));
     size_t line = m_textFile->GetCurrentLine();
     m_textFile->Close();
+#if wxUSE_UNICODE || wxCHECK_VERSION(2,3,2)
     m_textFile->Open(conv);
+#else
+    m_textFile->Open();
+#endif
     m_textFile->GoToLine(line);
 }
 
diff -ru poedit.orig/src/edframe.cpp poedit/src/edframe.cpp
--- poedit.orig/src/edframe.cpp	Mon Oct 29 00:04:34 2001
+++ poedit/src/edframe.cpp	Sun Nov 11 16:40:54 2001
@@ -1127,7 +1127,9 @@
     wxBusyCursor bcur;
     UpdateMenu();
 
+#if wxCHECK_VERSION(2,3,2)
     m_list->Freeze();
+#endif
     m_list->ClearAll();
     m_list->CreateColumns();
 
@@ -1141,7 +1143,9 @@
     AddItemsToList(*m_catalog, m_list, pos, 
                    CatFilterRest, g_ItemColourNormal);
 
+#if wxCHECK_VERSION(2,3,2)
     m_list->Thaw();
+#endif
     
     if (m_catalog->GetCount() > 0) 
         m_list->SetItemState(0, wxLIST_STATE_SELECTED, wxLIST_STATE_SELECTED);
diff -ru poedit.orig/src/manager.cpp poedit/src/manager.cpp
--- poedit.orig/src/manager.cpp	Sat Oct 27 18:35:26 2001
+++ poedit/src/manager.cpp	Sun Nov 11 16:41:26 2001
@@ -212,7 +212,9 @@
         wxDir::GetAllFiles(tkn.GetNextToken(), &m_catalogs, 
                            _T("*.po"), wxDIR_FILES);
 
+#if wxCHECK_VERSION(2,3,2)
     m_listCat->Freeze();
+#endif
 
     m_listCat->ClearAll();
     m_listCat->InsertColumn(0, _("Catalog"));
@@ -230,7 +232,9 @@
     m_listCat->SetColumnWidth(3, wxLIST_AUTOSIZE_USEHEADER);
     m_listCat->SetColumnWidth(4, wxLIST_AUTOSIZE);
     
+#if wxCHECK_VERSION(2,3,2)
     m_listCat->Thaw();
+#endif
 }
 
 
